File: Algorithms/Programs/C/Factorial.c
#include <stdio.h>

int main()
{
    int n;
    printf("Enter a number: ");
    scanf("%d", &n);
    int fact = 1;
    for (int i = 1; i <= n; i++)
    {
        fact *= i;
    }
    printf("Factorial of %d is %d", n, fact);
    return 0;
}

File: Algorithms/Programs/JavaScript/factorial.js
// Factorial using iterative approach
function factorial(num)
{
    if(num >= 0)
    {
        let fact = 1;
        for(let i = 1; i <= num; i++)
        {
            fact *= i;
        }
        return fact;
    }

    else return undefined;
}

console.log(factorial(any_number_here))